cpp/test/neighbors/ann_cagra/test_float_uint64_t.cu

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2023, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <gtest/gtest.h>
+
+#undef RAFT_EXPLICIT_INSTANTIATE_ONLY
+#include "../ann_cagra.cuh"
+
+namespace raft::neighbors::experimental::cagra {
+
+typedef AnnCagraTest<float, float, std::uint64_t> AnnCagraTestF_I64;
```

Comment:
Other ANN methods use `int64_t`, and it would be preferred to use the same here. Do we have any restrictions on signed vs non signed index types?

Reply:
I have changed it to `int64_t`.